data/tilesets/buildings/city_hall.license

```diff
@@ -0,0 +1,2 @@
+SPDX-License-Identifier: GPL-3.0-or-later
+SPDX-FileCopyrightText: 2019 Wahazar
```

Comment:
Not Wahazar's, first appeared [here](https://forum.freeciv.org/f/viewtopic.php?p=2151#p2151)

Reply:
thanks, good catch